examples/bri-3/src/bri/identity/bpiSubjects/models/bpiSubject.ts

```diff
@@ -2,6 +2,7 @@ import { IBpiSubject } from "./bpiSubject.interface";
 import { BpiSubjectType } from "./bpiSubjectType.enum";
 
 export class BpiSubject implements IBpiSubject {
+    // Fields of the domain model are always changed through it's methods
```

Comment:
maybe let's already make them private?

Reply:
we cant as long as we have interfaces as they do not allow for access modifiers.